### ISSUE TITLE
Feature request: per-run-configuration REPL "init"

### ISSUE THREAD

**Issue by michalmarczyk** [cursive-ide/cursive#1576]
Leiningen supports `:repl-options {:init (expression-to-be-evaluated-at-repl-startup)}`. This is quite convenient for setting up `require`s and such – one can have, say, a profile with `(require '[criterium.core :as c])` and some project-specific `require`s for benchmarking.

It'd be quite nice if Cursive could support something like this on a per-run-configuration basis – for non-Leiningen projects, for projects with many contributors no two of whom want the same configuration &c.

**Comment by p-himik** [comment]
I initially created #2187 because of lack of this option.
So another use-case is to run `in-ns` to simulate lein's `:init-ns`.

**Comment by otwieracz** [comment]
I feel like `lein` was created to solve those issues. Personal `profiles.clj` (kept out of VCS) and proper profiles feels much better than custom expressions you need to teach everyone to use when coming into project.

**Comment by p-himik** [comment]
@otwieracz As I mentioned in another issue, I don't use `lein`.
Apart from that, different people have different workflows. The fact that I work mainly in one namespace (or that the OP often uses `criterium` in REPL) doesn't mean that every other developer working with the same codebase does the same.

**Comment by otwieracz** [comment]
I am mentioning about `lein` only because it will probably resolve your issue sooner than IDE will take over part of build system job.

You probably have it

**Comment by p-himik** [comment]
Yes, I have it and still use it occasionally for some projects that are out of my control.
However, I have had some issues with it and its ecosystem - severe enough to make me not want to use it again any time soon.

**Comment by cursive-ide** [comment]
Yes, I'm planning to add this option - it also makes starting CLJS REPLs easier.
